### DISCUSSION TITLE
References do not show up in Vault QA mode

### DISCUSSION THREAD

**Discussion by devinat1** [logancyang/obsidian-copilot#544] [Q&A]
Prompt: What are the different types of web agents?
![image](https://github.com/logancyang/obsidian-copilot/assets/15788108/44ed4105-b8ff-4cd4-822d-ad97cacc95ef)

Response
```
In the context of web agents, which are software applications designed to act for a user or another program in a relationship of agency, there are several types, each with distinct functionalities and purposes. These agents navigate the World Wide Web, performing tasks, retrieving information, and interacting with web services on behalf of users or other programs. The different types of web agents include:

1. **Search Agents**: These agents specialize in finding information on the web based on user queries. They use algorithms to scour the web and return results that are most relevant to the user's search terms. Examples include traditional search engines like Google and Bing.

2. **Crawler Agents (or Spiders/Web Crawlers)**: These are a specific type of search agent that systematically browse the web to index the content of websites. This indexing is what allows search engines to return quick and relevant results to user queries. They are crucial for maintaining an up-to-date search index.

3. **Personal Agents**: Personal agents act on behalf of a specific user, performing tasks like managing emails, scheduling appointments, or even making recommendations for shopping. They learn from the user's preferences and actions to provide more personalized assistance over time.

4. **Monitoring and Surveillance Agents**: These agents keep track of changes to websites or monitor the web for specific information. They can alert users to changes in content, price fluctuations for products, or the appearance of specific terms or topics in news outlets or social media.

5. **Data Mining Agents**: These agents analyze vast amounts of data on the web to extract meaningful patterns, trends, and relationships. They are used in various fields, including market research, competitive analysis, and social media monitoring, to provide insights that would be difficult to discern manually.

6. **Transactional Agents**: These agents perform transactions on behalf of users, such as buying or selling stocks, booking flights, or purchasing goods online. They can automate the process of finding the best deals and completing the transaction process.

7. **Information Filtering Agents (or Recommender Systems)**: These agents filter through large amounts of information to present the user with content that matches their interests or preferences. They are commonly used in streaming services, e-commerce sites, and news aggregators to personalize the user experience.

8. **Navigation Agents**: Navigation agents assist users in navigating through complex websites or online services, helping them find the information or features they need more efficiently.

Each type of web agent plays a distinct role in how users interact with the vast resources of the internet, automating tasks, personalizing experiences, and managing information in ways that enhance productivity and accessibility.

**Sources**:
```

**Describe how to reproduce**
Ask any question in vault QA mode.
Use the OpenAI GPT4 Turbo

**Expected behavior**
Show sources that are the notes.

**Additional context**
SYSTEM INFO:
	Obsidian version: v1.5.12
	Installer version: v1.4.16
	Operating system: Windows 10 Home 10.0.26100
	Login status: logged in
	Catalyst license: none
	Insider build toggle: off
	Live preview: on
	Base theme: dark
	Community theme: Minimal v7.5.4
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 66
	Plugins enabled: 34
		1: Templater v2.2.3
		2: Tab Switcher v1.4.0
		3: Google Photos v1.6.5
		4: Dataview v0.5.66
		5: Editor Syntax Highlight v0.1.3
		6: Minimal Theme Settings v7.5.0
		7: Commander v0.5.1
		8: Canvas Mindmap v0.1.9
		9: Auto Link Title v1.5.3
		10: Admonition v10.2.0
		11: Simple CanvaSearch v0.1.0
		12: QuickAdd v1.8.1
		13: Excalidraw v2.1.3
		14: Todoist Sync v1.12.0
		15: Linter v1.23.2
		16: Readwise Official v2.0.1
		17: Enhancing Export v1.10.4
		18: Zotero Integration v3.1.7
		19: Paste URL into selection v1.7.0
		20: Note Refactor v1.8.2
		21: Canvas Filter v0.9.4
		22: Ghost Fade Focus v3.0.0
		23: WakaTime v1.0.0
		24: Quick Switcher++ v4.1.0
		25: Recent Files v1.3.10
		26: ExcaliBrain v0.2.15
		27: Projects v1.17.2
		28: Periodic Notes v0.0.17
		29: Homepage v3.8.0
		30: Calendar v1.5.10
		31: Longform v2.0.7
		32: BRAT v1.0.1
		33: Slides Extended v2.0.6
		34: Copilot v2.5.2

**Comment by logancyang** [comment]
Let me know if this is still an issue.

ps. when submitting an issue please screenshot your dev console with copilot debug mode on.
